docker/service/src/main/java/uk/nhs/adaptors/scr/config/ConversationIdFilter.java

```diff
@@ -40,15 +42,15 @@ protected void doFilterInternal(final HttpServletRequest request, final HttpServ
         }
     }
 
-    public void applyCorrelationId(String id) {
-        MDC.put(MDC_KEY, id);
-    }
-
     public String getRandomCorrelationId() {
         return UUID.randomUUID().toString().toUpperCase().replace("-", "");
```

Comment:
I think there is no need to remove dashes
https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir#api-Relatedperson-getRelatedPeople